Dynavity/Dynavity/view-model/canvas/CanvasViewModel.swift

```diff
@@ -22,4 +24,9 @@ class CanvasViewModel: ObservableObject {
         // Reset the selected image.
         selectedImage = nil
     }
+
+    func storeAnnotation(_ drawing: PKDrawing) {
```

Comment:
This function doesn't seem to be doing anything? Should `var` be removed?

Reply:
Whoops. You're right, I'll remove the `var`, thanks!